_docs/data-model/v1/upload.md

```diff
@@ -55,6 +59,48 @@ At some point, we may support real-time monitoring of devices, in which case the
 
 The Tidepool platform supports the idea that a user other than the data owner may have upload permissions (for example, a clinic or a parent). The byUser field should be the actual userId of the user doing the upload. This field will be stored in encrypted form in the actual data stream as it constitutes PII.
 
+## device information
+
+### deviceType
+This is the broad category of the device. In general, it should be one of the following:
+
+* `"pump"` (for insulin pumps)
+* `"cgm"` (for continuous glucose monitors)
+* `"pump-cgm"` (for pumps integrated with a cgm)
+* `"bgm"` (for blood glucose meters that are not continuous meters)
+* `"pen"` (for insulin injection devices)
+* `"manual"` (for manually entered data)
+
+For devices that do not fit in one of these categories, please contact Tidepool so we can standardize on a good name.
+
+### deviceManufacturer
+
+This is the name of the device manufacturer. We are trying to standardize this field as well, so please use exactly the following names:
+
+    Abbott
+    Asante
+    Dexcom
+    Insulet
+    Medtronic
+    OneTouch
```

Comment:
I struggled with this one. Even thought about putting the legal company name in ("Asante Solutions, Inc."). Maybe, since the brand is what you see when you buy these things, we should go for the brand instead. In which case Abbot->FreeStyle, OneTouch stays the same, and we should use MiniMed.